src/main/java/uk/ac/ebi/eva/pipeline/parameters/validation/ParametersValidatorUtil.java

```diff
@@ -28,6 +28,30 @@
  */
 public class ParametersValidatorUtil {
 
+    static void checkIsValidString(String stringToValidate,
+                                   String jobParametersName) throws JobParametersInvalidException {
+        checkIsNotNullOrEmptyString(stringToValidate, jobParametersName);
+        checkAsciiString(stringToValidate, jobParametersName);
+        checkSingleLineString(stringToValidate, jobParametersName);
+    }
+
+    static void checkAsciiString(String stringToValidate,
+                                 String jobParametersName) throws JobParametersInvalidException {
+        if (!stringToValidate.matches("\\A\\p{ASCII}*\\z")) {
```

Comment:
We should also support non-ASCII characters, like letters with tildes or dieresis. The ones we should reject are non-printable characters.